setup.py

```diff
@@ -20,8 +20,8 @@ def find_stub_files(name):
     readme = f.read()
 
 dependencies = [
-    "mypy>=0.790",
-    "django-stubs>=1.7.0",
+    "mypy>=0.930,<0.950",
```

Comment:
is this upper bound on mypy necessary?  (and does it relate to the purpose of this MR?)

This project releases rather infrequently.  So as a user I'd prefer that it did not pin me to a mypy version that will soon become stale.  

If and when mypy release a breaking change, I can always pin it myself - but the recent evidence (per the current requirement without an upper bound) is that that's probably not needed.

Reply:
Mypy can break its plugin API in new releases. We do the same in `django-stubs`

Reply:
Hmm, well if I'd known that I'd have asked you not to do it there too!

Perhaps you know something that I don't about mypy versioning, it seems mysterious to me.  Is there any reason to suspect that 0.950 would be breaking but 0.949 would not?

Reply:
> is this upper bound on mypy necessary? (and does it relate to the purpose of this MR?

I just set it to the same one as `django-stubs` as suggested in https://github.com/typeddjango/djangorestframework-stubs/pull/203#discussion_r835755300.